package.json

```diff
@@ -8,12 +8,12 @@
   },
   "dependencies": {
     "nes.css": "^2.3.0",
-    "vue": "^3.0.0-beta.15"
+    "vue": "^3.0.0-rc.5"
   },
   "devDependencies": {
-    "@vue/compiler-sfc": "^3.0.0-beta.15",
-    "eslint": "^7.2.0",
-    "eslint-plugin-vue": "^7.0.0-alpha.6",
-    "vite": "^0.20.0"
+    "@vue/compiler-sfc": "^3.0.0-rc.5",
+    "eslint": "^7.6.0",
+    "eslint-plugin-vue": "^7.0.0-beta.1",
+    "vite": "^0.20.10"
```

Comment:
Shall we go wild and bump it all the way to the 1.0.0 release candidate?

Probably got a load of breaking changes in it though 😂

Reply:
lets boost